mappings/net/minecraft/client/render/entity/model/WolfEntityModel.mapping

```diff
@@ -1,6 +1,16 @@
 CLASS net/minecraft/class_624 net/minecraft/client/render/entity/model/WolfEntityModel
+	FIELD field_20788 realHead Lnet/minecraft/class_630;
+	FIELD field_20789 realTail Lnet/minecraft/class_630;
+	FIELD field_27538 rightHindLeg Lnet/minecraft/class_630;
+	FIELD field_27539 leftHindLeg Lnet/minecraft/class_630;
+	FIELD field_27540 rightFrontLeg Lnet/minecraft/class_630;
+	FIELD field_27541 leftFrontLeg Lnet/minecraft/class_630;
 	FIELD field_3617 tail Lnet/minecraft/class_630;
+		COMMENT This is the main bone used to animate the tail. It contains {@link #real_tail} as one of its children.
```

Comment:
```suggestion
		COMMENT The main bone used to animate the tail. Contains {@link #realTail} as one of its children.
```

Reply:
Arguably worse than speaking in full sentences.

Reply:
The link target should still be changed tho

Reply:
Usually field comments use that format.